db/queries/scores/__init__.py

```diff
@@ -1,8 +1,8 @@
 # flake8: noqa
-from .queries import get_latest_score_for_application_sub_crit
-from .queries import create_score_for_application_sub_crit
+from .queries import get_scores_for_app_sub_crit
+from .queries import create_score_for_app_sub_crit
```

Comment:
I dont think we need these __init__ files for 'queries/scores' and 'queries/assessment_records'

Reply:
your right! i think there outer __init __ file handles it good spot! :D